test/integration/plugin_test.go

```diff
@@ -214,3 +217,211 @@ func TestPluginEssentials(t *testing.T) {
 		return expect404WithNoRoute(t, proxyURL.String(), resp)
 	}, ingressWait, waitTick)
 }
+
+func TestPluginOrdering(t *testing.T) {
+	if !util.GetKongVersion().GTE(semver.MustParse("3.0.0")) || kongEnterpriseEnabled == "" {
+		t.Skip("Kong version does not support plugin ordering")
+	}
+	t.Parallel()
+	ns, cleanup := namespace(t)
+	defer cleanup()
```

Comment:
We should probably take this opportunity to switch to the recently added `cluster.Cleaner` utility in KTF so that we're reducing noise for cleanup.